formstack-baton-requests/src/local/com/gu/identity/formstackbatonrequests/FormstackBatonLambdaLocalRun.scala

```diff
@@ -36,7 +42,42 @@ object FormstackBatonLambdaLocalRun extends App {
     val performSarLambdaConfig = FormstackConfig.getPerformHandlerConfig
     val performSarLambda = FormstackPerformSarHandler(Dynamo(), FormstackService, S3, performSarLambdaConfig)
     val streams = requestStreams(request)
-    performSarLambda.handleRequest(streams.inputStream, streams.outputStream)
+    performSarLambda.handleRequest(streams.inputStream, streams.outputStream, null)
+    val responseString = new String(streams.outputStream.toByteArray)
+    println("lambda output was: " + responseString)
+  }
+
+  def formstackRerTestRun(request: RerRequest): Unit = {
+    val rerLambdaConfig = FormstackConfig.getInitHandlerConfig
+    val rerLambda = FormstackSarHandler(S3, StepFunction, rerLambdaConfig)
+    val streams = requestStreams(request)
+    rerLambda.handleRequest(streams.inputStream, streams.outputStream, null)
+    val responseString = new String(streams.outputStream.toByteArray)
+    println("lambda output was: " + responseString)
+  }
+
+  def formstackPerformRerTestRun(request: RerRequest): Unit = {
+    val performRerLambdaConfig = FormstackConfig.getPerformHandlerConfig
+    val performRerLambda = FormstackPerformSarHandler(Dynamo(), FormstackService, S3, performRerLambdaConfig)
+    val streams = requestStreams(request)
+    performRerLambda.handleRequest(streams.inputStream, streams.outputStream, null)
+    val responseString = new String(streams.outputStream.toByteArray)
+    println("lambda output was: " + responseString)
+  }
+
+  private def requestUpdateStreams(request: UpdateDynamoRequest): InputOutputStreams = {
+    val jsonRequest = request.asJson.noSpaces
+    println(jsonRequest)
```

Comment:
may be we do not need this.

Reply:
> many things in this PR. looks good to me.

Yeah sorry about that. It needed more changes than I thought in order to achieve the changes for the step function. Thank you very much for looking at it. Hopefully this is the last big Baton-related PR from us!